src/server/v2.0/handler/project.go

```diff
@@ -63,3 +241,380 @@ func (a *projectAPI) GetLogs(ctx context.Context, params operation.GetLogsParams
 		WithLink(a.Links(ctx, params.HTTPRequest.URL, total, query.PageNumber, query.PageSize).String()).
 		WithPayload(auditLogs)
 }
+
+func (a *projectAPI) GetProject(ctx context.Context, params operation.GetProjectParams) middleware.Responder {
+	if err := a.RequireProjectAccess(ctx, params.ProjectID, rbac.ActionRead); err != nil {
+		return a.SendError(ctx, err)
+	}
+
+	p, err := a.getProject(ctx, params.ProjectID, project.WithCVEAllowlist(), project.WithOwner())
+	if err != nil {
+		return a.SendError(ctx, err)
+	}
+
+	return operation.NewGetProjectOK().WithPayload(model.NewProject(p).ToSwagger())
+}
+
+func (a *projectAPI) GetProjectDeletable(ctx context.Context, params operation.GetProjectDeletableParams) middleware.Responder {
+	if err := a.RequireProjectAccess(ctx, params.ProjectID, rbac.ActionDelete); err != nil {
+		return a.SendError(ctx, err)
+	}
+
+	result, err := a.deletable(ctx, params.ProjectID)
+	if err != nil {
+		return a.SendError(ctx, err)
+	}
+
+	return operation.NewGetProjectDeletableOK().WithPayload(result)
+}
+
+func (a *projectAPI) GetProjectSummary(ctx context.Context, params operation.GetProjectSummaryParams) middleware.Responder {
+	if err := a.RequireProjectAccess(ctx, params.ProjectID, rbac.ActionRead); err != nil {
+		return a.SendError(ctx, err)
+	}
+
+	p, err := a.getProject(ctx, params.ProjectID)
+	if err != nil {
+		return a.SendError(ctx, err)
+	}
+
+	summary := &models.ProjectSummary{
+		ChartCount: int64(p.ChartCount),
+		RepoCount:  p.RepoCount,
+	}
+
+	var fetchSummaries []func(context.Context, *project.Project, *models.ProjectSummary)
+
+	if hasPerm := a.HasProjectPermission(ctx, p.ProjectID, rbac.ActionRead, rbac.ResourceQuota); hasPerm {
+		fetchSummaries = append(fetchSummaries, getProjectQuotaSummary)
+	}
+
+	if hasPerm := a.HasProjectPermission(ctx, p.ProjectID, rbac.ActionList, rbac.ResourceMember); hasPerm {
+		fetchSummaries = append(fetchSummaries, getProjectMemberSummary)
```

Comment:
this will be goroutine in goroutine, I mean the mechod getProjectMemberSummary.

Reply:
Yes, you are right.